tests/controller_tests/resources/response_assertion_data.py

```diff
@@ -52,57 +63,57 @@
     ],
     "indirect_fund_unsecured": [
         {
-            "amount": 0,
+            "amount": 0.0,
             "comments": "",
             "current_status": "",
             "end_date": "2021-06-30T00:00:00Z",
             "funding_source_category": "Private Funding",
             "funding_source_name": "Private Funding",
-            "potential_secure_date": "",
+            "potential_secure_date": "NaTZ",
             "project_id": "FHSFDCC001",
             "start_date": "2021-04-01T00:00:00Z",
             "state": "Actual",
         },
         {
-            "amount": 1234567,
+            "amount": 1234567.0,
             "comments": "",
             "current_status": "",
             "end_date": "2022-06-30T00:00:00Z",
             "funding_source_category": "Private Funding",
             "funding_source_name": "Private Funding",
-            "potential_secure_date": "",
+            "potential_secure_date": "NaTZ",
             "project_id": "FHSFDCC001",
             "start_date": "2022-04-01T00:00:00Z",
             "state": "Forecast",
         },
     ],
     "outcome_data": [
         {
-            "amount": -0.87,
+            "amount": 0,
             "end_date": "2020-04-30T00:00:00Z",
             "geography_indicator": "Town",
             "outcome": "Year on Year % Change in Monthly Footfall",
             "outcome_category": "Place",
             "project_id": "FHSFDCC001",
             "start_date": "2020-04-01T00:00:00Z",
             "state": "Actual",
-            "unit_of_measurement": "Year on Year % Change in Monthly Footfall",
+            "unit_of_measurement": "Year on Year % Change in Monthly " "Footfall",
```

Comment:
small point again but theres some ill formatting from copying from a test diff throughout this file

Reply:
what goes around comes around